src/main/java/com/afornalik/ox/board/Board.java

```diff
@@ -0,0 +1,35 @@
+package com.afornalik.ox.board;
+
+import java.util.Map;
+
+public class Board {
+
+    private Map<Integer, BoardField> boardFields;
+    private final int bound;
+
+    public Board(int bound) {
+        this.bound = bound;
+    }
+
+    public int getBound() {
+        return bound;
+    }
+
+    boolean initializeAllField(Map<Integer, BoardField> mapImplementation) {
+        int fieldQuantity = bound * bound;
+        boardFields = mapImplementation;
+        for (int i = 0; i < fieldQuantity; i++) {
+            boardFields.put(i, new BoardField());
+        }
```

Comment:
not needed in non-static map

Reply:
Deleted - i don't initialize map with boardField object so i don't allocate memory but i return null when i receiveBoardField